src/manifests_workflow/input_manifests.py

```diff
@@ -120,3 +130,24 @@ def write_manifest(self, version, components=[]):
         manifest_path = os.path.join(manifest_dir, f"{self.prefix}-{version}.yml")
         manifest.to_file(manifest_path)
         logging.info(f"Wrote {manifest_path}")
+
+    def add_to_cron(self, version):
+        logging.info(f"Adding new version to cron: {version}")
+        jenkinsfile = self.cron_jenkinsfile()
+        with open(jenkinsfile, "r") as f:
+            data = f.read()
+
+        cron_entry = f"H/10 * * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml;TARGET_JOB_NAME=distribution-build-{self.prefix}\n"
```

Comment:
I am worried that the hardcoding of H/10 here will be issue later.
I am more in favor of just adding the cron to jenkinsfile, since it is just one line for each build.

Reply:
The cron is to check for changes, takes a few seconds to run.

What are you worried about? We can always change this value, too.

Reply:
I think I get what you trying to do here.
Thanks.